content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -402,7 +402,7 @@ metadata:
 spec:
   tls:
   - hosts:
-    - sslexample.foo.com
+      - sslexample.foo.com
```

Comment:
Hello @marcobellaccini .
Just checking on this change:
`rules` and `tls` belong at the same scope?
And `hosts` and `host` belong at the same scope?

Reply:
Hi,
`rules` and `tls` are both parts of `spec` (see [IngressSpec doc](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#ingressspec-v1beta1-extensions)).
`hosts` and `host` belong, respectively, to the `tls` and `rules` nodes (see [IngressTLS](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#ingresstls-v1beta1-extensions) and [IngressRule](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#ingressrule-v1beta1-extensions) docs).
There were two problems with this example: a syntactical one and an aesthetical one.
The former resulted in a YML-parser error and I fixed it in https://github.com/kubernetes/website/pull/20828/commits/602177e6f937fe27d30876e6a42898567ee74e89.
The problem springs from the fact that, as stated in [YML spec](https://yaml.org/spec/1.2/spec.html#id2799784):
> nested block sequences may be indented by one less space to compensate, **except, of course, if nested inside another block sequence**.

Hence, I had to add two whitespaces to fix the indentation.

The latter problem, instead, was purely aesthetical: the `rules` and `tls` collections were not using the same style of indentation (in `tls` the dashes were considered part of the indentation while, in `rules`, they were not).
This did not result in any YML-parsing error, nevertheless I've fixed it in https://github.com/kubernetes/website/pull/20828/commits/bcea96792662f24089e6623aee0d2578f44588f2.

Reply:
Great. Thanks for fixing!